Misc/NEWS.d/next/Library/2022-12-10-08-36-07.gh-issue-100133.g-zQlp.rst

```diff
@@ -0,0 +1 @@
+Fix bug in :mod:`asyncio` where a subprocess would sometimes lose data received from pipe. Patch by Kumar Aditya.
```

Comment:
```suggestion
Fix regression in :mod:`asyncio` where a subprocess would sometimes lose data received from pipe. Patch by Kumar Aditya.
```
You could possibly drop the attribution of the patch (since you also broke it :-).